### ISSUE TITLE
Crash while loading / unloading a sample from Samples project

### ISSUE THREAD

**Issue by jetXS** [TouchDesigner/TouchEngine-UE4#106]
A bit random, and I was loading/unloading 2-3 samples at a time. Not easy to reproduce reliably.

```
Assertion failed: !IsGarbageCollecting() [File:D:/Build/++UE4/Sync/Engine/Source/Runtime/CoreUObject/Private/UObject/UObjectGlobals.cpp] [Line: 2367] Unable to create new object: Texture2D /Engine/Transient.None. Creating UObjects while Collecting Garbage is not allowed!

UE4Editor_Core
UE4Editor_Core
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor_Engine
UE4Editor_TouchEngine!<lambda_91251fb4a09ab15f063d950871354fad>::operator()() [C:\Users\workshop\_SOURCE\TouchEngine-UE4-Samples\Plugins\TouchEngine-UE4\Source\TouchEngine\Private\UTouchEngine.cpp:443]
UE4Editor_TouchEngine!TEnqueueUniqueRenderCommandType<`UTouchEngine::LinkValueCallback'::`17'::voidName,<lambda_91251fb4a09ab15f063d950871354fad> >::DoTask() [C:\Program Files\Epic Games\UE_4.27\Engine\Source\Runtime\RenderCore\Public\RenderingThread.h:183]
UE4Editor_TouchEngine!TGraphTask<TEnqueueUniqueRenderCommandType<`UTouchEngine::LinkValueCallback'::`17'::voidName,<lambda_91251fb4a09ab15f063d950871354fad> > >::ExecuteTask() [C:\Program Files\Epic Games\UE_4.27\Engine\Source\Runtime\Core\Public\Async\TaskGraphInterfaces.h:886]
UE4Editor_Core
UE4Editor_Core
UE4Editor_RenderCore
UE4Editor_RenderCore
UE4Editor_Core
UE4Editor_Core
kernel32
ntdll
```

**Comment by ddthejj** [comment]
This should be fixed in bbecd2889fa57bfa9383b3e802b0caa5d3877cf7
It's not the most elegant solution ever, but it works

**Comment by jetXS** [comment]
Can't reproduce anymore so considering it fixed for now. Thanks.
